python/trifinger_simulation/finger_types_data.py

```diff
@@ -32,6 +32,10 @@ def get_valid_finger_types():
     Returns:
         list: List of supported finger types.
     """
+    print("Among these listed finger types, 'single' and 'tri'"
+          " are deprecated. Use 'fingerone' and 'trifingerone'"
+          " instead respectively."
+            )
```

Comment:
I'm still strongly against this.  If it is a regular print now, it will be even more annoying.

I'm not sure where this is used (if it would break things) but how about we simply remove these entries from the list before returning it?

Reply:
I think this is more of a utitlity method that the user may call once before actually doing anything. And "single" and "tri" are still valid finger types. So, this message will only get printed once anyway.